editor/animation_track_editor.cpp

```diff
@@ -6142,10 +6142,10 @@ AnimationTrackEditor::AnimationTrackEditor() {
 	edit->get_popup()->add_item(TTR("Scale Selection"), EDIT_SCALE_SELECTION);
 	edit->get_popup()->add_item(TTR("Scale From Cursor"), EDIT_SCALE_FROM_CURSOR);
 	edit->get_popup()->add_separator();
-	edit->get_popup()->add_shortcut(ED_SHORTCUT("animation_editor/duplicate_selection", TTR("Duplicate Selection"), KeyModifierMask::CMD | Key::D), EDIT_DUPLICATE_SELECTION);
+	edit->get_popup()->add_shortcut(ED_SHORTCUT("animation_editor/duplicate_selection", TTR("Duplicate Selected"), KeyModifierMask::CMD | Key::D), EDIT_DUPLICATE_SELECTION);
```

Comment:
I'm not sure if the previous form was incorrect. Either way, if you intend to change this, the shortcut code should be updated as well (`animation_editor/duplicate_selected`).

Reply:
Well my original concern was about the Delete Selection, but then I noticed that duplicate could be changed too. "Delete Selection" implies that the selection will be deleted (whatever that means) instead of what is selected. Or maybe it's just me 🤔

Reply:
"selection" does mean "the things which are selected", so that was correct as is IMO: https://en.wiktionary.org/wiki/selection

For example in VS Code it has "Run Selection in ..." so it's also referring to the actual contents of the selected text buffer:
![Screenshot_20211123_104731](https://user-images.githubusercontent.com/4701338/143002481-e7e8a099-fd04-446d-a2c8-395026cb135b.png)